commit 6d981f348e9522a33184bc009682a87b05034f09
Author: Lukas Rusak <lorusak@gmail.com>
Date:   Thu Oct 27 00:05:11 2016 -0700

    enet add install to CMakeLists.txt, add PIC

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1fe2f1b..8f15260 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -52,6 +52,7 @@ endif()
  
 include_directories(${PROJECT_SOURCE_DIR}/include)
  
+set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 add_library(enet STATIC
         callbacks.c
         compress.c
@@ -63,3 +64,15 @@ add_library(enet STATIC
         unix.c
         win32.c
     )
+
+# Install the generated static library
+install(TARGETS ${PROJECT_NAME}
+        ARCHIVE DESTINATION ${CMAKE_INSTALL_PREFIX}/lib)
+
+# Install the headers
+install(DIRECTORY include/enet
+        DESTINATION ${CMAKE_INSTALL_PREFIX}/include)
+
+# Install the enet-config.cmake script
+install(FILES cmake/enet-config.cmake
+        DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/cmake/enet)
diff --git a/cmake/enet-config.cmake b/cmake/enet-config.cmake
new file mode 100644
index 0000000..c95f230
--- /dev/null
+++ b/cmake/enet-config.cmake
@@ -0,0 +1,2 @@
+find_path(ENET_INCLUDE_DIRS enet  HINTS "/usr/include")
+find_library(ENET_LIBRARIES NAMES libenet.a HINTS "/usr/lib")
