From d030678787cd91096e9271ddba87acb59b611361 Mon Sep 17 00:00:00 2001
From: Garrett Brown <themagnificentmrb@gmail.com>
Date: Thu, 14 Jul 2016 19:46:35 -0700
Subject: [PATCH] [cmake] Add missing install scripts and set PIC to ON

---
 CMakeLists.txt | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1fe2f1b..88e36a2 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -52,6 +52,7 @@ endif()
  
 include_directories(${PROJECT_SOURCE_DIR}/include)
  
+set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 add_library(enet STATIC
         callbacks.c
         compress.c
@@ -63,3 +64,16 @@ add_library(enet STATIC
         unix.c
         win32.c
     )
+
+# Install the generated static library
+install(TARGETS ${PROJECT_NAME}
+        ARCHIVE DESTINATION ${CMAKE_INSTALL_PREFIX}/lib)
+
+# Install the headers
+install(DIRECTORY include/enet
+        DESTINATION ${CMAKE_INSTALL_PREFIX}/include)
+
+# Write the enet-config.cmake script
+file(WRITE ${CMAKE_INSTALL_PREFIX}/${PROJECT_NAME}-config.cmake
+     "set(ENET_LIB ${CMAKE_INSTALL_PREFIX}/lib/${CMAKE_STATIC_LIBRARY_PREFIX}${PROJECT_NAME}${CMAKE_STATIC_LIBRARY_SUFFIX})")
+
-- 
2.7.4

